### PR TITLE
Add a layer for pairwise application of bilinear forms

### PR DESCRIPTION
**Pull request by danieldk**
Finally a small PR ;).

### CODE REVIEW

File: sticker2/src/model/pairwise_bilinear.rs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+        assert_eq!(
+            input1.dim(),
+            3,
+            "Shape should have 3 dimensions, has: {}",
+            input1.dim()
+        );
+
+        // The shapes of the inputs are [batch_size, max_seq_len, features].
+        // After matrix multiplication, we the intermediate shape
```
Comment (DiveFish) [inline sticker2/src/model/pairwise_bilinear.rs]: 'we get/obtain'

File: sticker2/src/model/pairwise_bilinear.rs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        // [batch_size, max_seq_len, features].
+        let intermediate = input1.matmul(&self.weight);
+
+        // Transpose the second input to obtain the shape
+        // [batch_size, features, max_seq_len]. The perform a matrix
```
Comment (DiveFish) [inline sticker2/src/model/pairwise_bilinear.rs]: 'We perform'

**Review by DiveFish — COMMENTED** [review]
Just typos

**Comment by danieldk** [comment]
Thanks! Both are fixed.

**Review by DiveFish — APPROVED** [review]

**Pull request merged**